README.md

```diff
@@ -934,6 +934,20 @@ clusters to be UC compatible.
 
 [[back to top](#databricks-labs-ucx)]
 
+## `lint-local-code` command
+
+```text
+databricks labs ucx lint-local-code
+```
+
+Once [table migration](#table-migration-workflow) is complete, you can run this command to 
```

Comment:
Does it require the table migration to have run? It could also run before, right? I would recommend users to do so

Reply:
done